packages/flutter/lib/src/material/circle_avatar.dart

```diff
@@ -85,48 +93,113 @@ class CircleAvatar extends StatelessWidget {
   /// The size of the avatar. Changing the radius will cause the
   /// avatar to animate to the new size.
   ///
+  /// If radius is specified, then [minRadius] and [maxRadius] may not be
+  /// specified. Specifying radius is equivalent to specifying a [minRadius] and
+  /// [maxRadius], both with the value of radius.
+  ///
   /// Defaults to 20 logical pixels.
   final double radius;
 
+  /// The minimum size of the avatar.
+  ///
+  /// Changing the minRadius may cause the avatar to animate to the new size, if
+  /// constraints allow.
+  ///
+  /// If minRadius is specified, then [radius] may not also be specified.
+  ///
+  /// Defaults to zero.
+  final double minRadius;
+
+  /// The maximum size of the avatar.
+  ///
+  /// Changing the maxRadius will cause the avatar to animate to the new size,
+  /// if constraints allow.
+  ///
+  /// If maxRadius is specified, then [radius] may not also be specified.
+  ///
+  /// Defaults to [double.infinity].
+  final double maxRadius;
+
+  static const double _defaultRadius = 20.0;
```

Comment:
_kConstants

Reply:
Not for member constants, only globals. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k